retry-function.mk

```diff
@@ -0,0 +1,17 @@
+define retrycmd_if_failure
```

Comment:
yeah, I'd probably prefer having a new bash script parameterized with envrionment variables we call directly from the make file which can run cleanup/scan/test, which would include the retry logic in-line